bcda/constants/constants.go

```diff
@@ -1,6 +1,10 @@
-package testConstants
+package constants
 
 const DEVACOUUID = "0c527d2e-2e8a-4808-b11d-0fa06baf8254"
 const SMALLACOUUID = "3461C774-B48F-11E8-96F8-529269fb1459"
 const MEDIUMACOUUID = "C74C008D-42F8-4ED9-BF88-CEE659C7F692"
 const LARGEACOUUID = "8D80925A-027E-43DD-8AED-9A501CC4CD91"
+
+// App Name and usage.  Edit them here to prevent breaking tests
+const Name = "bcda"
+const Usage = "Beneficiary Claims Data API CLI"
```

Comment:
Because they're for the CLI (and not really used elsewhere), `Name` and `Usage` might make more sense in the `bcdacli` package.

Reply:
How strongly do feel about this change?  I moved them because I feel like we shouldn't be isolating our constants to just the scope where they are used.  By putting all constants in constants.go we create a single source of truth for all constants regardless of where they exist in the code and make it easier in the event they are used elsewhere in the future.

Reply:
If we are moving them to a centralized location, maybe we should name them more specific?  A general constant named `Name` and `Usage` makes it hard to tell what it would be used for... if we want to keep these constants in a common file, maybe we should name them something like `App_Name` and `App_Usage` or `CLI_Name` and `CLI_Usage`?

Reply:
I'm not a fan in general of putting all constants in one place. To me it makes a lot more sense to have them in the package they apply to. No matter where they're used, the package works as a descriptor. E.g., `bcdacli.Name` is the name used by the CLI.

Reply:
@embh convinced me.  I have put it back in bcdacli